syntax = "proto3";

package SS;

import "proto/client_server.proto";

message ForwardedMessage {
  string sender_id = 1;
  client_server.Message message = 2;
}

service ForwardingService {
  rpc Forward(ForwardedMessage) returns (client_server.Empty);
}